streams/us_a1x.m3u

```diff
@@ -0,0 +1,13 @@
+#EXTM3U
+#EXTINF:-1 tvg-id="WABCDT1.us",WABC-DT1 (720p)
+https://a1xs.vip/300003
+#EXTINF:-1 tvg-id="MSNBC.us",MSNBC (720p)
+https://a1xs.vip/300008
```

Comment:
Redirecting to a Xtream-Codes server...

http://cdn1host.online:2999/mariobackup/rLyzlSWd6l/143

![image](https://github.com/user-attachments/assets/c2b60afb-aab0-4713-aa1f-6333a838732c)